redash/tasks/queries.py

```diff
@@ -243,7 +244,9 @@ def truncate_long_string(original_str, max_length):
 
 
 @celery.task(name="redash.tasks.get_table_sample_data")
-def get_table_sample_data(existing_columns, data_source_id, table_name, table_id):
+def get_table_sample_data(data_source_id, table_name, table_id):
```

Comment:
Please:

1. Name this `update_sample`
2. Add a docstring describing what this does